ydb/library/yql/minikql/comp_nodes/mkql_wide_top_sort.cpp

```diff
@@ -637,8 +589,47 @@ using TBase = TComputationValue<TSpillingSupportState<Sort, HasCount>>;
         return true;
     }
 
+    NUdf::TUnboxedValue* ExtractInMemory() {
+        if (Full.empty())
+            return nullptr;
+
+        const auto ptr = Full.back();
+        Full.pop_back();
+        return static_cast<NUdf::TUnboxedValue*>(ptr);
+    }
+
+    void SealInMemory() {
+        if constexpr (!HasCount) {
+            static_assert (Sort);
+            // Remove placeholder for new data
+            Storage.resize(Storage.size() - Indexes.size());
+
+            Full.reserve(Storage.size() / Indexes.size());
```

Comment:
Do we have guarantees that `Indexes.size()` is not 0?

Reply:
Yes. Indexes size is the number of input columns to the callable.